### PR TITLE
Copying input files to run directories #3

### PR DESCRIPTION
**Pull request by kshitij-v-mehta**

### CODE REVIEW

File: codar/cheetah/model.py [hunk 1]
```diff
@@ -62,6 +63,9 @@ def __init__(self, machine_name, app_dir):
         self.machine = self._get_machine(machine_name)
         self.app_dir = os.path.abspath(app_dir)
         self.runs = []
+        for input in self.inputs:
```
Comment (bd4) [inline codar/cheetah/model.py]: `input` is a built-in function, maybe call it `input_rpath`?

Comment (kshitij-v-mehta) [inline codar/cheetah/model.py]: PR updated.

File: codar/cheetah/launchers.py [hunk 2]
```diff
@@ -245,6 +246,9 @@ def write_batch_script(self, runs):
                 # TODO: abstract this to higher levels
                 os.makedirs(run.run_path, exist_ok=True)
 
+                for input in run.inputs:
```
Comment (bd4) [inline codar/cheetah/launchers.py]: here too

File: codar/cheetah/model.py [hunk 3]
```diff
@@ -43,6 +43,7 @@ class Campaign(object):
     codes = {}
     supported_machines = []
     sweeps = []
+    inputs_fullpath = []
```
Comment (bd4) [inline codar/cheetah/model.py]: I think `inputs = []` is also needed, for Campaign definitions that don't have any inputs.

Comment (kshitij-v-mehta) [inline codar/cheetah/model.py]: you are right. PR updated.

Comment (kshitij-v-mehta) [inline codar/cheetah/model.py]: I have tested this now with both heat_map and Pi examples.

**Pull request merged**